src/app/src/actions/facilities.js

```diff
@@ -31,7 +36,24 @@ export function fetchFacilities() {
 
         return apiRequest
             .get(makeGetFacilitiesURLWithQueryString(qs))
-            .then(({ data }) => dispatch(completeFetchFacilities(data)))
+            .then(({ data }) => {
+                const responseHasOnlyOneFacility = get(
+                    data,
+                    'features',
+                    [],
+                ).length === 1;
+
+                if (responseHasOnlyOneFacility) {
+                    flow(
```

Comment:
Sounds good -- this is in 7327ad107b83153417172ce4dccc25e18ac7307b

Since the code is in a promise the existing `catch` will handle the thrown error.